codegen/smithy-aws-kotlin-codegen/src/test/kotlin/software/amazon/smithy/kotlin/codegen/aws/protocols/RpcV2CborTest.kt

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package software.amazon.smithy.kotlin.codegen.aws.protocols
+
+import software.amazon.smithy.kotlin.codegen.test.*
+import kotlin.test.Test
+
+class RpcV2CborTest {
+    val model = """
+        ${"$"}version: "2"
+
+        namespace com.test
+        
+        use smithy.protocols#rpcv2Cbor
+        use aws.api#service
+
+        @rpcv2Cbor
+        @service(sdkId: "CborExample")
+        service CborExample {
+            version: "1.0.0",
+            operations: [GetFoo, GetFooStreaming]
+        }
+
+        @http(method: "POST", uri: "/foo")
```

Comment:
Small nit, feel free to ignore it: This looks like it should be `"GET"` instead of `"POST"`

Reply:
Same thing for the `GetFooStreaming` operation